airflow/models.py

```diff
@@ -1092,7 +1152,7 @@ def error(self, session=None):
         session.commit()
 
     @provide_session
-    def refresh_from_db(self, session=None, lock_for_update=False, include_queue_time=False):
+    def refresh_from_db(self, session=None, lock_for_update=False, include_queue_time=False, include_job_id=False):
```

Comment:
I'm continuing in our storied tradition of only including new fields out of refresh_from_db if they're explicitly called for.  I instigated this pattern when I discovered (to my horror) that simply always pulling all the fields from the database when asked to update a TaskInstance will break parts of airflow.  Frankly I'm afraid to investigate why, and we don't really have time, so we have some patented _cowardly params_ ™️  to help us out.